.circleci/config.yml

```diff
@@ -291,7 +291,22 @@ commands:
           command : |
             curl --data '{"text": "❌ Job *'$CIRCLE_JOB'* failed on *'$CIRCLE_BRANCH'*. Please see '$CIRCLE_BUILD_URL' for details."}' "$SLACK_WEBHOOK_URL"
           when: on_fail
-
+  run-rpc:
```

Comment:
It could be more clear if you separate "preparing" and "executing" parts of this command. 
For example, we can define a command `prepare-rpc-node` with the same processes except for the last line (line 309). And then, we can add the last line in `test-rpc` and `rpc-tester-report` as a `run: command:`.